pkg/providers/instancetype/nodetemplate_test.go

```diff
@@ -530,6 +530,65 @@ var _ = Describe("NodeTemplate/InstanceTypes", func() {
 		}
 		Expect(nodeNames.Len()).To(Equal(1))
 	})
+	It("should launch instances for AWS NeuronCore resource requests", func() {
+		nodeNames := sets.NewString()
+		ExpectApplied(ctx, env.Client, provisioner, nodeTemplate)
+		pods := []*v1.Pod{
+			coretest.UnschedulablePod(coretest.PodOptions{
+				ResourceRequirements: v1.ResourceRequirements{
+					Requests: v1.ResourceList{v1alpha1.ResourceAWSNeuronCore: resource.MustParse("4")},
+					Limits:   v1.ResourceList{v1alpha1.ResourceAWSNeuronCore: resource.MustParse("4")},
```

Comment:
Should we have a test to handle launching an instance with inf1 since this is the only instance that has `* 4` on the neuron cores compared to neuron devices. Maybe a testcase that requests a high number of neuroncores and can only schedule on inf1?

Reply:
Its now actually tied into this test since one of the pods requests 64 neuroncores (only satisfiable by `inf1.24xlarge` instance types.